spec/spec_helper.rb

```diff
@@ -19,6 +19,7 @@
 require 'database_cleaner'
 require 'ffaker'
 require 'vcr'
```

Comment:
No commit message, this is also simple enough to just add these dependencies in the commit that requires it.